batch7crud-roll1/src/main/webapp/resources/css/layout.css

```diff
@@ -0,0 +1,54 @@
+.wrapper{
+	width:100%; 
+}
+
+.header{
+	width: 100%;
+	min-height: 60px;
+	background-color: #00CA79;
+}
+
+.header-content{
+	padding: 0 20px;
+}
+
+.company-name{
+	width: 40%;
+	padding-top: 15px;
+	float: left;
+}
+
+.main-body{
+	width: 100%;
+	min-height: 600px;
+	background-color: #E2E2E2;
+}
+
+.sidebar{
+	width: 20%;
+	float: left;
+	min-height: 600px;
+	background-color: #333333;
+}
+
+.content-wrapper{
+	width: 80%;
+	float: right;
+	min-height: inherit;
+    background-color: #E2E2E2
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a semicolon at the end of this declaration [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Asemicolon-declaration)